template.yml

```diff
@@ -251,10 +251,19 @@ Resources:
       ContentUri: org-code-javabuilder/change_runtime_directory
       Description: Change Java runtime to launch from the writeable /tmp directory to enable student projects to write files more easily.
       LayerName: change-java-runtime-directory
+  AddFontConfigLayer:
```

Comment:
@sureshc tagged you to look at template changes

Reply:
Maybe name this `FontConfigurationLayer`? CloudFormation templates are declarative, so we don't need to use the verb `Add`.